napari/_qt/dialogs/_tests/test_qt_about_key_bindings.py

```diff
@@ -55,7 +55,7 @@ def test_updating_with_layer_change(make_napari_viewer, monkeypatch):
     # add a shape layer (different keybindings)
     viewer.add_shapes(None, shape_type='polygon')
     # check that the new layer is the active_layer
-    assert viewer.active_layer == viewer.layers[1]
+    assert viewer.layers.selection.active == viewer.layers[1]
```

Comment:
@jni This is the other most consequential end user changes

Reply:
this is one where I think we might want to provide an alias at `viewer.layers.active`

Reply:
Yeah I'm into either of those ideas... Having said that it's not necessary to get that done in this PR if it's only going to be an alias, right?

Reply:
yep, it can wait.